tfx/orchestration/portable/launcher.py

```diff
@@ -133,7 +134,7 @@ def _register_execution(
       exec_properties=exec_properties)
 
 
-class Launcher:
+class Launcher(object):
```

Comment:
Changes in this file seems not needed any more.

Reply:
Fixed this.